pyproject.toml

```diff
@@ -1,10 +1,3 @@
 [tool.black]
 line-length = 100
 target-version = ['py38', 'py39']
-
-[tool.isort]
```

Comment:
This should stay in the configuration, so it doesn't change in the future.

Reply:
I'm doing all the configuration in #169.